.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Type of change (place an x in the [ ] that applies)
+
+- [ ] New application
+- [ ] Bug fix (non-breaking change which fixes an issue) 
+- [ ] New feature (non-breaking change which adds functionality) 
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected) 
+- [ ] Documentation update
+
+### Summary
+
+Describe the goal of this PR. Mention any related Issue numbers. 
+
+If you are introducing a new sample app in this pull request, please also include a detailed description of:
+
+- The application’s purpose and main functionality
+- What framework(s), languages, and technologies the application uses
```

Comment:
```suggestion
If you are introducing a new sample app in this pull request, please also include a detailed description of the application’s purpose and functionality.
```

The framework, language and dependencies should be fairly obvious, so no need for them to mention it explicitly.